app/domains/fluggastrechte/formular/stringReplacements.ts

```diff
@@ -117,3 +118,21 @@ export const getWeiterePersonenNameStrings = (
         context.weiterePersonen?.[arrayIndex].nachname,
     };
 };
+
+export const getResponsibleCourt = (context: FluggastrechtContext) => {
+  const court = getCourtByStartAndEndAirport(
+    context.startAirport || "",
```

Comment:
I think null coalescing is recommended by linter
```suggestion
    context.startAirport ?? "",
```